src/engine/UIManager.cxx

```diff
@@ -160,16 +160,24 @@ void UIManager::init()
           dynamic_cast<Button *> (uiElement.get())->isToggleButton = toggleButton;
           dynamic_cast<Button *> (uiElement.get())->drawImageButtonFrame(drawFrame);
           break;
-        case ElementType::TextButton:
-          uiElement = std::make_unique<Button>(elementRect);
-          dynamic_cast<Button *>(uiElement.get())->setText(text);
-          dynamic_cast<Button *>(uiElement.get())->isToggleButton = toggleButton;
+        case ElementType::TextButton: {
+          int fontSize = element.value("FontSize", Settings::instance().getFontSize(uiElementType));
+          auto elmButton = std::make_unique<Button>(elementRect);
+          elmButton->setText(text);
+          elmButton->isToggleButton = toggleButton;
+          elmButton->setFontSize(fontSize);
+          uiElement.reset(elmButton.release());
```

Comment:
```suggestion
          uiElement = std::move(elmButton);
```

Looks less convoluted. Same for line 178

Reply:
ok